src/views/Home/components/Banners/IFOBanner.tsx

```diff
@@ -86,15 +94,43 @@ const RightWrapper = styled.div`
   }
 `
 
+const activeIfo = ifosConfig.find((ifo) => ifo.isActive)
+
 const IFOBanner = () => {
   const { t } = useTranslation()
+  const currentBlock = useCurrentBlock()
+
+  const { data: currentIfoBlocks = { startBlock: 0, endBlock: 0 } } = useSWRImmutable(
+    activeIfo ? ['ifo', 'currentIfo'] : null,
+    async () => {
+      const abi = activeIfo.version === 3.1 ? ifoV3Abi : ifoV2Abi
+      const [startBlock, endBlock] = await multicallv2(
+        abi,
+        [
+          {
+            address: activeIfo.address,
+            name: 'startBlock',
+          },
+          {
+            address: activeIfo.address,
+            name: 'endBlock',
+          },
+        ],
+        { requireSuccess: false },
+      )
+
+      return { startBlock: startBlock ?? 0, endBlock: endBlock ?? 0 }
+    },
+  )
+
+  const isIfoAlive = !!(activeIfo && currentBlock && currentIfoBlocks && currentIfoBlocks.endBlock > currentBlock)
 
-  return (
+  return isIfoAlive ? (
     <Wrapper>
       <Inner>
         <LeftWrapper>
-          <StyledSubheading>{t('Live')}</StyledSubheading>
-          <StyledHeading scale="xl">ERA IFO</StyledHeading>
+          <StyledSubheading>{currentIfoBlocks.startBlock < currentBlock ? t('Live') : t('Soon')}</StyledSubheading>
```

Comment:
<img width="669" alt="Screen Shot 2022-02-21 at 16 34 22" src="https://user-images.githubusercontent.com/97418926/154927719-f1489f59-cc0c-4e5d-ab4d-3175562adcce.png">

Could we reuse this function to get status ?

Reply:
Done